addons/interaction/functions/fnc_renameGroup.sqf

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+/*
+ * Author: Seb
+ * Renames a group to a given string (groupID), whilst checking that it is not an invalid name.
+ *
+ * Arguments:
+ * 0: The group to be renamed <GROUP>
+ * 1: The new name of the group <STRING>
+ *
+ * Return Value:
+ * Whether the group was succesfully renamed <BOOL>
+ *
+ * Example:
+ * [group player, "leet squad"] call ace_interaction_fnc_renameGroup
+ *
+ * Public: No
+ */
+
+params [
+    ["_group", grpNull, [grpNull]],
+    ["_newName", "", [""]]
+];
+if (_newName isEqualTo (groupID _group)) exitWith {true};
+
+//private _nameAlreadyTaken = (toLower _newName) in (allGroups apply {toLower (groupID _x)});
+private _lowerName = toLower _newName; // Stops "Alpha 1-1" and "alpha 1-1" being compatible names etc.
+private _nameAlreadyTaken = allGroups findIf {
+    side _x isEqualTo side _group
+    && {_lowerName isEqualTo toLower (groupID _x)}
+} != -1;
+
+
+if (_nameAlreadyTaken) then {
+    hint LLSTRING(RenameGroupAlreadyExists);
+} else {
+    _group setGroupIdGlobal [_newName];
```

Comment:
```suggestion
    [_group, _newName] call CBA_fnc_setCallsign;
```
Just stumbled over this. This cba function could be utalized instead of the `setGroupIdGlobal`. Does the same basicly but next frame.

https://github.com/CBATeam/CBA_A3/blob/92bf7f0a62955f8b2a8ed35bae60e2360e91bb00/addons/common/fnc_setCallsign.sqf

Reply:
That command doesn't seem to do anything my code isn't already doing, seems rather pointless to implement it.